### PR TITLE
Add daedalus-flight 2.2.0

### PR DESCRIPTION
**Pull request by yottalogical**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask-versions/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-versions/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

**Comment by miccal** [comment]
Would this conflict with [`daedalus-mainnet`](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/daedalus-mainnet.rb)?

**Comment by yottalogical** [comment]
> Would this conflict with [`daedalus-mainnet`](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/daedalus-mainnet.rb)?

No. `daedalus-mainnet` and `daedalus-flight` are designed to be able to run side-by-side without conflicting with one another.

File: Casks/daedalus-flight.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+cask "daedalus-flight" do
+  version "2.2.0,14238"
+  sha256 "395e0a57da910903127cc6aaaa793df42edc375e7423ead1df90a870c2f28116"
+
+  # update-cardano-mainnet-flight.iohk.io/ was verified as official when first introduced to the cask
+  url "https://update-cardano-mainnet-flight.iohk.io/daedalus-#{version.before_comma}-mainnet_flight-#{version.after_comma}.pkg"
+  name "Daedalus Flight"
```
Comment (miccal) [inline Casks/daedalus-flight.rb:7]: Please add a description, perhaps something like:
```
desc "Cryptocurrency wallet"
```

Comment (yottalogical) [reply Casks/daedalus-flight.rb:7]: The `desc` stanza has been added.

**Comment by vitorgalvao** [comment]
Seems to be the equivalent to a beta, so please submit to [homebrew/cask-versions](https://github.com/Homebrew/homebrew-cask-versions).

**Comment by miccal** [comment]
@vitorgalvao this _is_ the `homebrew-cask-versions` repo.

**Comment by vitorgalvao** [comment]
> @vitorgalvao this _is_ the `homebrew-cask-versions` repo.

I really hate the new Github Notifications page. Not the first time I’ve made this mistake. Thank you.

**Comment by miccal** [comment]
> I really hate the new Github Notifications page. Not the first time I’ve made this mistake. Thank you.

Agreed, and you are welcome.

**Comment by vitorgalvao** [comment]
Thank you for the submission @yottalogical, and apologies for the mistaken close.

**Pull request merged**